testsuite/domain/pom.xml

```diff
@@ -177,7 +177,6 @@
                     <argLine>${jvm.args.ip.client} ${jvm.args.timeouts} ${surefire.system.args}</argLine>
 
                     <systemPropertyVariables>
-                        <jboss.options>-ea ${surefire.system.args}</jboss.options>
```

Comment:
This is somewhat an oversight, only -ea should have been removed as it is by default now present in surefire.system.args already. parameter itself should stay.